src/Symfony/Installer/NewCommand.php

```diff
@@ -137,7 +137,7 @@ private function checkSymfonyVersionIsInstallable()
             return $this;
         }
 
-        // validate server syntax
+        // validate semver syntax
```

Comment:
@xabbuh your proposal is easier to understand ... but any developer looking at the source code of this command probably knows what `semver` is :)